sw-injector-template.js

```diff
@@ -78,8 +78,17 @@ const swCancelCacheRoutes = () => {
 	}, 1000);
 };
 
+/**
+ * Ask the service worker to remove itself.
+ */
+const swClearCache = () => {
```

Comment:
I might elect for a more descriptive name, like ``swClearCachesUninstall``, but even that doesn't encompass the cleaning of the db... I think it is fine, so this isn't really a request, just a _thought_.